test/acceptance/send_test.go

```diff
@@ -138,10 +139,11 @@ receivers:
 	// Times are provided in fractions of seconds.
 	am.Push(At(1), Alert("alertname", "test").Active(1))
 
-	at.Do(At(1.2), func() {
-		am.Terminate()
-		am.Start()
-	})
+	// XXX(fabxc): disabled as long as alerts are not persisted.
+	// at.Do(At(1.2), func() {
+	//	am.Terminate()
+	//	am.Start()
+	// })
```

Comment:
Should we just remove this commented out code? It can always be pulled back out of Git's history when alerts are persisted again.